### ISSUE TITLE
audio_recorder: 1.0.2 ios build error

### ISSUE THREAD

**Issue by codesniper9315** [ZaraclaJ/audio_recorder#64]
Hello, guys!
Currently, I am using audio_recorder package for my flutter projects.
I am just facing issues when I try to build ios app.

ISSUE: 'AVAudioSessionCategoryOptions' has been renamed to 'AVAudioSession.CategoryOptions
I tried to build project with another version but still face same issue.
How can I fix that issue permantely?

Looking forward.
Thanks

**Comment by codesniper9315** [comment]
Is there any solution?

**Comment by gyakhoe** [comment]
I'm facing the same issue. I think the pull request should to sent to it or we have to use another package.

**Comment by gyakhoe** [comment]
> Is there any solution?

I don't think we have solution right now. I am also working on voice recorder app. I think it's better to use another package. I tried applying the changes to swift file but kept on getting more error. I fixed those and more error appeared. I don't think it is worth the time as of now.

**Comment by gyakhoe** [comment]
> Is there any solution?

consider checking out my voice recorder app build using
[https://github.com/gyakhoe/voice_recorder.git](url)
flutter_audio_recorder and audio_players
